### PR TITLE
Integrate Trello to supply the restaurant list

### PR DESCRIPTION
**Pull request by dispatchrabbi**

### CODE REVIEW

**Comment by dispatchrabbi** [comment]
You will need to `npm install` for this PR.

File: partials/restaurant-info.html
Comment (kristinstaton) [inline partials/restaurant-info.html:5]: This change eliminates the link to reservations, which is totally fine, if that's what you intended. If we do want it to show up, we'd need to add a custom field to Trello for it.

**Comment by kristinstaton** [comment]
Before I merge this, will you eyeball my comment about the removal of the reservations link?

File: partials/restaurant-info.html
Comment (dispatchrabbi) [reply partials/restaurant-info.html:5]: Correct. I figure we can add it back in pretty simply if we need to.

**Comment by dispatchrabbi** [comment]
Generally the person who _makes_ the PR is the one who merges it, after comments from reviewers. (It means they can make sure they got everything, or if a comment spurs something that should be in the PR that the commenter doesn't realize, the submitter gets a chance to put that in.)

**Comment by kristinstaton** [comment]
Well, then, I feel really dumb asking, but what's the correct next step here? Doesn't this get automatically merged once I approve the changes (files changed > review changes > approve) if the status right ^^ up there says "Merging can be performed automatically with one approved review." Is it just a matter of me not using the right language to describe what I think the next step is?

**Comment by dispatchrabbi** [comment]
It shouldn't. It should just green light it and wait for someone to
manually merge. Approve the PR and let's see what happens!

**Review by kristinstaton — APPROVED** [review]

**Comment by kristinstaton** [comment]
Weeeeeeeird. Lucky me, nothing is broken!

**Pull request merged**
